### PR TITLE
update basic features for 7.1

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Update basic features for v7.1

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/13740
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -20,218 +20,218 @@ This document lists the features supported in different TiDB versions, including
...
-| [Optimizer hints](/optimizer-hints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
-| [MPP execution engine](/explain-mpp.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | N |
-| [MPP execution engine - compression exchange](/explain-mpp.md#mpp-version-and-exchange-data-compression) | Y | Y | N | N | N | N | N | N | N | N |
-| [Index Merge](/explain-index-merge.md) | Y | Y | Y | Y | Y | E | E | E | E | E |
-| [Placement Rules in SQL](/placement-rules-in-sql.md) | Y | Y | Y | Y | E | E | N | N | N | N |
-| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) | E | E | E | E | E | E | E | E | E | E |
+| Advanced SQL features | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Prepared statement cache](/sql-prepared-plan-cache.md) | Y | Y | Y | Y | Y | E | E | E | E |
+| [Non-prepared statement cache](/sql-non-prepared-plan-cache.md) | E | N | N | N | N | N | N | N | N |
```
Comment (Oreoxmt) [inline basic-features.md:94]:
```suggestion
| [Non-prepared statement cache](/sql-non-prepared-plan-cache.md) | Y | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 2]
```diff
@@ -20,218 +20,218 @@ This document lists the features supported in different TiDB versions, including
...
+| [Titan Level Merge](/storage-engine/titan-configuration.md#level-merge-experimental) | E | E | E | E | E | E | E | E | E |
+| [Use buckets to improve scan concurrency](/tune-region-performance.md#use-bucket-to-increase-concurrency) | E | E | E | N | N | N | N | N | N |
+| [Invisible indexes](/sql-statements/sql-statement-add-index.md) | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [Composite `PRIMARY KEY`](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Unique indexes](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Clustered index on integer `PRIMARY KEY`](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Clustered index on composite or non-integer key](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [Multi-valued index](/sql-statements/sql-statement-create-index.md#multi-valued-index) | E | N | N | N | N | N | N | N | N |
+| [Foreign key](/constraints.md#foreign-key) | Y | N | N | N | N | N | N | N | N |
+| [TiFlash late materialization](/tiflash/tiflash-late-materialization.md) | E | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:63]:
```suggestion
| [TiFlash late materialization](/tiflash/tiflash-late-materialization.md) | Y | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 3]
```diff
@@ -20,218 +20,218 @@ This document lists the features supported in different TiDB versions, including
...
+| [Common Table Expressions (CTE)](/sql-statements/sql-statement-with.md) | Y | Y | Y | Y | Y | Y | Y | N | N |
+| `START TRANSACTION`, `COMMIT`, `ROLLBACK` | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [`EXPLAIN`](/sql-statements/sql-statement-explain.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [User-defined variables](/user-defined-variables.md) | E | E | E | E | E | E | E | E | E |
+| [`BATCH [ON COLUMN] LIMIT INTEGER DELETE`](/sql-statements/sql-statement-batch.md) | Y | Y | Y | N | N | N | N | N | N |
+| [`BATCH [ON COLUMN] LIMIT INTEGER INSERT/UPDATE/REPLACE`](/sql-statements/sql-statement-batch.md) | Y | Y | N | N | N | N | N | N | N |
+| [`ALTER TABLE ... COMPACT`](/sql-statements/sql-statement-alter-table-compact.md) | Y | Y | E | N | N | N | N | N | N |
+| [Table Lock](/sql-statements/sql-statement-lock-tables-and-unlock-tables.md) | E | E | E | E | E | E | E | E | E |
+| [TiFlash Query Result Materialization](/tiflash/tiflash-results-materialization.md) | E | E | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:87]:
```suggestion
| [TiFlash Query Result Materialization](/tiflash/tiflash-results-materialization.md) | Y | E | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 4]
```diff
@@ -20,218 +20,218 @@ This document lists the features supported in different TiDB versions, including
...
-| [Range INTERVAL partitioning](/partitioned-table.md#range-interval-partitioning) | E | E | E | N | N | N | N | N | N | N |
+| Partitioning | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Range partitioning](/partitioned-table.md#range-partitioning) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Hash partitioning](/partitioned-table.md#hash-partitioning) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Key partitioning](/partitioned-table.md#key-partitioning) | Y | N | N | N | N | N | N | N | N |
+| [List partitioning](/partitioned-table.md#list-partitioning) | Y | Y | Y | E | E | E | E | E | N |
+| [List COLUMNS partitioning](/partitioned-table.md) | Y | Y | Y | E | E | E | E | E | N |
+| [`EXCHANGE PARTITION`](/partitioned-table.md) | Y | Y | E | E | E | E | E | E | N |
+| [`REORGANIZE PARTITION`](/partitioned-table.md#reorganize-partitions) | Y | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:150]:
```suggestion
| [`REORGANIZE PARTITION`](/partitioned-table.md#reorganize-partitions) | Y | N | N | N | N | N | N | N | N |
| [`COALESCE PARTITION`](/partitioned-table.md#decrease-the-number-of-partitions) | Y | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 5]
```diff
@@ -20,218 +20,218 @@ This document lists the features supported in different TiDB versions, including
...
+| [Dynamic pruning](/partitioned-table.md#dynamic-pruning-mode) | Y | Y | Y | E | E | E | E | N | N |
+| [Range COLUMNS partitioning](/partitioned-table.md#range-columns-partitioning) | Y | Y | N | N | N | N | N | N | N |
+| [Range INTERVAL partitioning](/partitioned-table.md#range-interval-partitioning) | E | E | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:153]:
```suggestion
| [Range INTERVAL partitioning](/partitioned-table.md#range-interval-partitioning) | Y | E | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 6]
```diff
@@ -20,218 +20,219 @@ This document lists the features supported in different TiDB versions, including
...
-| [Change data capture (CDC)](/ticdc/ticdc-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
-| [Stream data to Amazon S3, Azure Blob Storage, NFS, and GCP through TiCDC](/ticdc/ticdc-sink-to-cloud-storage.md) | Y | E | E | N | N | N | N | N | N | N |
-| [TiCDC supports bidirectional replication between two TiDB clusters](/ticdc/ticdc-bidirectional-replication.md) | Y | Y | Y | N | N | N | N | N | N | N |
-| [TiCDC OpenAPI v2](/ticdc/ticdc-open-api-v2.md) | Y | N | N | N | N | N | N | N | N | N |
+| Data import and export | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Fast Importer (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| mydumper logical dumper | Deprecated | Deprecated | Deprecated | Deprecated | Deprecated | Deprecated | Deprecated | Deprecated | Deprecated |
+| [Dumpling logical dumper](/dumpling-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Transactional `LOAD DATA`](/sql-statements/sql-statement-load-data.md) | E [^5] | Y | Y | Y | Y | Y | Y | Y | N [^6] |
```
Comment (hfxsd) [inline basic-features.md:196]: @Frank945946 PTAL
```suggestion
| [Transactional `LOAD DATA`](/sql-statements/sql-statement-load-data.md) | Y [^5] | Y | Y | Y | Y | Y | Y | Y | N [^6] |
```

File: basic-features.md [hunk 7]
```diff
@@ -20,218 +20,219 @@ This document lists the features supported in different TiDB versions, including
...
+| [RocksDB engine](/storage-engine/rocksdb-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Titan plugin](/storage-engine/titan-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Titan Level Merge](/storage-engine/titan-configuration.md#level-merge-experimental) | E | E | E | E | E | E | E | E | E |
+| [Use buckets to improve scan concurrency](/tune-region-performance.md#use-bucket-to-increase-concurrency) | E | E | E | N | N | N | N | N | N |
+| [Invisible indexes](/sql-statements/sql-statement-add-index.md) | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [Composite `PRIMARY KEY`](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Unique indexes](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Clustered index on integer `PRIMARY KEY`](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Clustered index on composite or non-integer key](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [Multi-valued index](/sql-statements/sql-statement-create-index.md#multi-valued-index) | E | N | N | N | N | N | N | N | N |
```
Comment (ran-huang) [inline basic-features.md:61]:
```suggestion
| [Multi-valued index](/sql-statements/sql-statement-create-index.md#multi-valued-index) | Y | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 8]
```diff
@@ -20,218 +20,219 @@ This document lists the features supported in different TiDB versions, including
...
-| [Change column type](/sql-statements/sql-statement-modify-column.md) | Y | Y | Y | Y | Y | Y | Y | Y | N | N |
-| [Temporary tables](/temporary-tables.md) | Y | Y | Y | Y | Y | Y | N | N | N | N |
-| Concurrent DDL statements | Y | Y | Y | N | N | N | N | N | N | N |
-| [Acceleration of `ADD INDEX` and `CREATE INDEX`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) | Y | Y | Y | N | N | N | N | N | N | N |
-| [Metadata lock](/metadata-lock.md) | Y | Y | Y | N | N | N | N | N | N | N |
-| [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) | Y | Y | Y | N | N | N | N | N | N | N |
+| Data definition language (DDL) | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| Basic `CREATE`, `DROP`, `ALTER`, `RENAME`, `TRUNCATE` | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Generated columns](/generated-columns.md) | E | E | E | E | E | E | E | E | E |
```
Comment (ran-huang) [inline basic-features.md:113]:
```suggestion
| [Generated columns](/generated-columns.md) | Y | E | E | E | E | E | E | E | E |
```

File: basic-features.md [hunk 9]
```diff
@@ -20,218 +20,219 @@ This document lists the features supported in different TiDB versions, including
...
+| [Auto random](/auto-random.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TTL (Time to Live)](/time-to-live.md) | Y | E | N | N | N | N | N | N | N |
+| [DDL algorithm assertions](/sql-statements/sql-statement-alter-table.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| Multi-schema change: add columns | Y | Y | E | E | E | E | E | E | E |
+| [Change column type](/sql-statements/sql-statement-modify-column.md) | Y | Y | Y | Y | Y | Y | Y | N | N |
+| [Temporary tables](/temporary-tables.md) | Y | Y | Y | Y | Y | N | N | N | N |
+| Concurrent DDL statements | Y | Y | N | N | N | N | N | N | N |
+| [Acceleration of `ADD INDEX` and `CREATE INDEX`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) | Y | Y | N | N | N | N | N | N | N |
+| [Metadata lock](/metadata-lock.md) | Y | Y | N | N | N | N | N | N | N |
+| [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) | Y | Y | N | N | N | N | N | N | N |
```
Comment (ran-huang) [inline basic-features.md:127]:
```suggestion
| [Pause/Resume DDL](/ddl-introduction.md#ddl-related-commands) | E | N | N | N | N | N | N | N | N |

```

**Review by ran-huang — APPROVED** [review]

File: basic-features.md [hunk 10]
```diff
@@ -241,6 +243,6 @@ This document lists the features supported in different TiDB versions, including
 
 [^4]: Starting from [v6.4.0](/releases/release-6.4.0.md), TiDB supports [high-performance and globally monotonic `AUTO_INCREMENT` columns](/auto-increment.md#mysql-compatibility-mode)
 
-[^5]: For [TiDB v7.0.0](/releases/release-7.0.0.md), the new clauses `FORMAT`, `FIELDS DEFINED NULL BY`, `With batch_size=<number>,detached`, and support for importing data from S3 and GCS, are experimental features.
+[^5]: For [TiDB v7.1.0](/releases/release-7.0.0.md), integrating with TiDB Lightning Physical Import Mode and supporting import concurrencyare are experimental features.
```
Comment (Oreoxmt) [inline basic-features.md:246]:
```suggestion
[^5]: For [TiDB v7.1.0](/releases/release-7.1.0.md), integrating with TiDB Lightning Physical Import Mode and supporting import concurrency are experimental features.
```

File: basic-features.md [hunk 11]
```diff
@@ -20,218 +20,220 @@ This document lists the features supported in different TiDB versions, including
...
+| [`SET CONFIG`](/dynamic-config.md) | Y | Y | Y | E | E | E | E | E | E |
+| [DM WebUI](/dm/dm-webui-guide.md) | E | E | E | N | N | N | N | N | N |
+| [Foreground Quota Limiter](/tikv-configuration-file.md#foreground-quota-limiter) | Y | Y | E | N | N | N | N | N | N |
+| [Background Quota Limiter](/tikv-configuration-file.md#background-quota-limiter) | E | E | N | N | N | N | N | N | N |
+| [EBS volume snapshot backup and restore](https://docs.pingcap.com/tidb-in-kubernetes/v1.4/backup-to-aws-s3-by-snapshot) | Y | Y | N | N | N | N | N | N | N |
+| [PITR](/br/backup-and-restore-overview.md) | Y | Y | N | N | N | N | N | N | N |
+| [Global memory control](/configure-memory-usage.md#configure-the-memory-usage-threshold-of-a-tidb-server-instance) | Y | Y | N | N | N | N | N | N | N |
+| [Cross-cluster RawKV replication](/tikv-configuration-file.md#api-version-new-in-v610) | E | E | N | N | N | N | N | N | N |
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-new-in-v50) | E | E | E | E | E | E | E | E | N |
+| [Resource control](/tidb-resource-control.md) | E | N | N | N | N | N | N | N | N |
```
Comment (Oreoxmt) [inline basic-features.md:235]:
```suggestion
| [Resource control](/tidb-resource-control.md) | Y | N | N | N | N | N | N | N | N |
```

**Review by ran-huang — APPROVED** [review]

File: basic-features.md [hunk 12]
```diff
@@ -20,218 +20,220 @@ This document lists the features supported in different TiDB versions, including
...
-| [Acceleration of `ADD INDEX` and `CREATE INDEX`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) | Y | Y | Y | N | N | N | N | N | N | N |
-| [Metadata lock](/metadata-lock.md) | Y | Y | Y | N | N | N | N | N | N | N |
-| [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) | Y | Y | Y | N | N | N | N | N | N | N |
+| Data definition language (DDL) | 7.1 (upcoming) | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| Basic `CREATE`, `DROP`, `ALTER`, `RENAME`, `TRUNCATE` | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Generated columns](/generated-columns.md) | Y | E | E | E | E | E | E | E | E |
+| [Views](/views.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Sequences](/sql-statements/sql-statement-create-sequence.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Auto increment](/auto-increment.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
```
Comment (Oreoxmt) [inline basic-features.md:116]:
```suggestion
| [Auto increment](/auto-increment.md) | Y | Y[^4] | Y | Y | Y | Y | Y | Y | Y |
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**